_posts/2017-10-23-pair-programming-with-perry.md

```diff
@@ -0,0 +1,96 @@
+---
+layout: post
+authors:
+    - justinSchoen
+date: 2017-10-23 3:00:00 -0500
+title: Pair Programming with Perry
+---
+
+Hey, everyone. Today I’m going to interview Perry, a pair programmer practitioner, about pair programming, his experiences, and he’ll give us some tips on pairing effectively.
+
+<hr/>
+<span class='os-green'>*So, Perry, tell me, why should anyone pair program?*</span>
+<div style='margin-left: 2em;' >
+    Great question, you’ve earned a schrute buck for your curiosity, and yes, those are transferrable for Stanley nickels. Currency aside, here’s a common scenario: you can’t get unstuck on a problem so you seek someone’s help and in explaining the problem to them the solution comes to you, sometimes the other person doesn’t even say anything to help. This isn’t a new concept and it’s the reason you’ll sometimes hear people talk about rubber-duck programming, which is when you explain your code line by line to a rubber duck with intentions of discovering bugs and confusing logic, or getting unstuck.
```

Comment:
Here's a quick pass at this paragraph:
####################
Great question, you’ve earned a Schrute buck for your curiosity! Currency aside, here’s a common scenario: You're stuck on a specific problem while coding so you ask a fellow developer for help. When you start explaining the problem to them, the solution comes to you in a flash of brilliance, sometimes even without the other person saying a single word!

This concept is not new: It's called rubber-duck programming, and it means explaining your code, line by line, to a rubber duck. The intent is to slow your brain down and think through the issue, step by step, so that you can methodically uncover bugs and troubleshoot issues more methodically.

Reply:
Maybe change the interviewer's line to "So, Perry, tell me, why should anyone consider pair programming?